wagtail/tests/test_page_queryset.py

```diff
@@ -919,6 +919,78 @@ def test_specific_query_with_annotation(self):
         self.assertEqual(results.first().subscribers_count, 1)
         self.assertEqual(results.last().subscribers_count, 1)
 
+    def test_specific_subquery_select_related(self):
+        pages = list(
```

Comment:
Suggestion: Given the querysets are force evaluated here, it would be good to get some `assertNumQueries` calls here to confirm the number of queries executed is correct.

Reply:
Agreed. Will add!